### ISSUE TITLE
Need simple config option to support cookieconsent location:true

### ISSUE THREAD

**Issue by localegirl** [tinesoft/ngx-cookieconsent#57]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x ] feature request
```

### CookieConsent and Library Versions?
```
- cookieconsent version: 3.1.1
- ngx-cookieconsent version: 2.2.2
```

### OS Version?

MacOS Mojave

### Angular, Node and al Versions?

Angular CLI: 6.2.3
Node: 11.14.0
Angular: 6.1.9

### Repro steps

N/A

### The log given by the failure

N/A

### Desired functionality

https://cookieconsent.osano.com/documentation/javascript-api/ indicates that you can specify location:true in the configuration to easily enable the display of the popup *only* in countries that require it (and hide in others).

ngx-cookieconsent does not appear to support this, since the Typescript compiler requires an NgcLocationOptions object.

Is there an easy way to configure this? If not, would it be possible to add support for it?

**Comment by tinesoft** [comment]
Hi @localegirl,

This is a valid request. Thanks for reporting 👍 
I will update the typings tonight to allow this.

Expect a new feature release to come out, hopefully tonight


Cheers!

**Comment by localegirl** [comment]
Great! Thanks!

**Comment by tinesoft** [comment]
Released under [v2.2.3](https://github.com/tinesoft/ngx-cookieconsent/releases/tag/v2.2.3)

Enjoy! :)

**Comment by localegirl** [comment]
Awesome! Thanks for the quick response!
